### DISCUSSION TITLE
Can use Form builder in Custom Template (Blade)

### DISCUSSION THREAD

**Discussion by hoangnamitc** [orchidsoftware/platform#2337] [General]
**Is your feature request related to a problem? Please describe.**
I am using [Custom template](https://orchid.software/en/docs/custom-template/) to create Layout view.
but instead of typing HTML code, can i call Form builder method to quickly create Elements?
eg:
// ... /views/hello.blade.php 
```
{{ TextArea::make('description') }}
```

**Describe the solution you'd like**
...

**Describe alternatives you've considered**
...

**Additional context**
...

**Comment by tabuna** [comment]
Yes you can do it. For example, we do this in templates where we do not have data binding:  https://github.com/orchidsoftware/platform/blob/15bd88dcd688f8c69a22d568ea09452a42b56f14/resources/views/auth/signin.blade.php#L7

Your example does not work correctly due to escaping. If you want to display html, then use different brackets:

```php
{!! \Orchid\Screen\Fields\TextArea::make('description') !!}}
```

**Reply by hoangnamitc** [reply]
Thanks Answer !
